redis/src/cluster_routing.rs

```diff
@@ -428,6 +434,81 @@ impl RoutingInfo {
                 r.arg_idx(streams_position + 1)
                     .map(|key| RoutingInfo::for_key(cmd, key))
             }
+
+            // keyless commands with more arguments, whose arguments might be wrongly taken to be keys.
+            // TODO - double check these, in order to find better ways to route some of them.
+            b"ACL DRYRUN"
+            | b"ACL GENPASS"
+            | b"ACL GETUSER"
+            | b"ACL HELP"
+            | b"ACL LIST"
+            | b"ACL LOG"
+            | b"ACL USERS"
+            | b"ACL WHOAMI"
+            | b"AUTH"
+            | b"TIME"
+            | b"PUBSUB CHANNELS"
+            | b"PUBSUB NUMPAT"
+            | b"PUBSUB NUMSUB"
+            | b"PUBSUB SHARDCHANNELS"
+            | b"BGSAVE"
+            | b"WAITAOF"
+            | b"SAVE"
+            | b"LASTSAVE"
+            | b"CLIENT TRACKINGINFO"
+            | b"CLIENT PAUSE"
+            | b"CLIENT UNPAUSE"
+            | b"CLIENT UNBLOCK"
+            | b"CLIENT ID"
+            | b"CLIENT REPLY"
+            | b"CLIENT GETREDIR"
+            | b"CLIENT INFO"
+            | b"CLIENT KILL"
+            | b"READONLY"
+            | b"CLUSTER INFO"
+            | b"CLUSTER MEET"
+            | b"CLUSTER MYSHARDID"
+            | b"CLUSTER NODES"
+            | b"CLUSTER REPLICAS"
+            | b"CLUSTER RESET"
+            | b"CLUSTER SET-CONFIG-EPOCH"
+            | b"CLUSTER SLOTS"
+            | b"CLUSTER SHARDS"
+            | b"CLUSTER COUNT-FAILURE-REPORTS"
+            | b"CLUSTER KEYSLOT"
+            | b"COMMAND"
+            | b"COMMAND COUNT"
+            | b"COMMAND LIST"
+            | b"COMMAND GETKEYS"
+            | b"CONFIG GET"
+            | b"DEBUG"
+            | b"READWRITE"
+            | b"TFUNCTION LOAD"
+            | b"TFUNCTION DELETE"
+            | b"TFUNCTION LIST"
+            | b"TFCALL"
+            | b"TFCALLASYNC"
+            | b"MODULE LIST"
+            | b"MODULE LOAD"
+            | b"MODULE UNLOAD"
+            | b"CLIENT GETNAME"
```

Comment:
can you order the lists so it would be easier to iterate through the list - or just to move client getname up with the other client commands

Reply:
done